migration/migrator/migrations/system/20190125144723_cpp_argument_parsing.py

```diff
@@ -12,6 +12,7 @@ def up(config):
     os.system("tar -xpzf {0}".format(TCLAP_TAR))
     os.remove(TCLAP_TAR)
     os.chdir(TCLAP_DIR)
+    os.system("sed -i 's/SUBDIRS = include examples docs tests msc config/SUBDIRS = include docs msc config/' Makefile.in")
```

Comment:
Are we re-running this migration as right now this line won't have any effect on the live system (which had this migration run today).

We could also just rollback both the python migration and this one and then re-execute this one if we wanted without any danger.

Reply:
That shouldn't be necessary. This change just makes the migration run faster by stopping it from compiling TCLAP's built in tests. Besides a speedup, nothing else should change.